Project/Source/Resources/ResourceMaterial.cpp

```diff
@@ -372,6 +373,8 @@ void ResourceMaterial::OnEditorUpdate() {
 
 	ImGui::NewLine();
 	if (ImGui::Button("Save##material")) {
-		SaveToFile(GetResourceFilePath().c_str());
+		if (FileDialog::GetFileExtension(GetAssetFilePath().c_str()) != DEFAULT_MODEL_EXTENSION) {
```

Comment:
MODEL_EXTENSION is thowing undeclared identifier

Reply:
Fixed